src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -2026,7 +2026,11 @@ public override BoundNode VisitWhileStatement(BoundWhileStatement node)
 
         public override BoundNode VisitWithExpression(BoundWithExpression node)
         {
-            // PROTOTYPE: This is wrong
+            VisitRvalue(node.Receiver);
+            foreach (var (_, expr) in node.Arguments)
```

Comment:
>foreach [](start = 12, length = 7)

Note that Andy's latest PR changes the structure of bound nodes. You'll want to rebase on to of it.
Then consider `return VisitObjectOrCollectionInitializerExpression(node.InitializerExpression);`